packages/ffe-form-react/src/PhoneNumberField.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {string, bool, func} from 'prop-types';
+
+import i18n from './i18n/i18n';
+
+export default function PhoneNumberField({
+                               phone,
+                               countryCode,
+                               onChange,
+                               onBlur,
+                               locale,
+                               disabled,
+                               showValidationError,
+                               className
+                           }) {
+
+    const supportedLocales = ['nb', 'nn', 'en'];
+    const text = i18n[supportedLocales.indexOf(locale) !== -1 ? locale : 'nb'];
+
+    function onPhoneChange(e) {
+        onChange(countryCode, e.target.value);
+    }
+
+    function onCountryCodeChange(e) {
+        onChange(e.target.value, phone);
+    }
+
+    function handleBlur(e) {
+        if (e.target.id !== 'country-code' || e.target.value.length === 0) {
+            onBlur(e);
+        }
+    }
+
+    return (
+        <div className={`ffe-input-group ${className ? className : 'phone-number-field'}`}>
+            <div className="phone-number-field">
+                <div className="phone-number-field__country-code">
+                    <label className="ffe-form-label" htmlFor="country-code">{text.COUNTRY_CODE}</label>
+                    <div className="phone-number-field__input-group">
+                        <span className="phone-number-field__plus">+</span>
+                        <input
+                            id="country-code"
+                            maxLength="3"
+                            className="ffe-input-field phone-number-field__country-code-input"
+                            type="tel"
+                            disabled={disabled}
+                            value={countryCode}
+                            aria-invalid={showValidationError}
+                            onChange={onCountryCodeChange}
+                            onBlur={handleBlur}
+                        />
+                    </div>
+                </div>
+                <div className="phone-number-field__number">
+                    <label className="ffe-form-label" htmlFor="phone-input">{text.PHONE_NUMBER}</label>
+                    <input
+                        id="phone-input"
+                        maxLength="14"
+                        type="tel"
+                        className="ffe-input-field phone-number-field__phone-input"
+                        onChange={onPhoneChange}
+                        onBlur={onBlur}
+                        value={phone}
+                        aria-invalid={showValidationError}
```

Comment:
It could probably just forward all the props it doesn't care about to the `<input>` element. That way a consumer of this component could write 
```xml
<PhoneNumberField
    aria-invalid={true}
    maxLength="14"
/>
```
And then the component wouldn't need to have opinions about maxLength either.

We could just do
```javascript
    const {
        countryCode,
        onChange,
        onBlur,
        locale,
        className,
        ...rest
    } = props;
```
And then send `rest` to the input. That way we don't have to manually pass stuff like the `disabled` prop either. `value` could also be passed this way instead of being aliased as `phone`? I feel that would be more in line with [not renaming DOM props](https://github.com/SpareBank1/designsystem/blob/develop/CONTRIBUTING.md#videresend-props).

Reply:
Agree completely - but this is slightly complicated by the fact that there are two input fields. 

- Should you be able to add `aria-invalid` to the country code input? 
- Should the properties be applied to both?
- Should you be able to disable one field. but not the other?
- How do you deal with change listeners?

Finally, if we're going to have some kind of wrapping DOM element - how should we forward props to that element?